backend/tests/integration/database/test_migrations_integration.py

```
@@ -0,0 +1,214 @@
+            assert len(result) == 1, "Should have exactly one version record"
+
+
+@pytest.mark.integration
+class TestMigrationErrorCases:
+    """Test error handling in migration scenarios."""
+
+    def test_migration_with_invalid_database_url(self) -> None:
+        """Test that invalid database URLs are handled properly."""
+        invalid_url = "invalid://not/a/real/database"
+
+        with pytest.raises(Exception):
```

Ruff (0.13.2)

Do not assert blind exception: `Exception`

(B017)